.circleci/git_decrypt.sh

```diff
@@ -1,12 +1,5 @@
-# Install git-crypt
-cd
-git clone https://github.com/AGWA/git-crypt.git
-cd git-crypt
-make
-sudo make install
-
 # Unlock encrypted files
 cd ~/refinebio/.circleci
-openssl aes-256-cbc -d -in exported.key.enc -out $KEY_FILENAME -k $OPENSSL_KEY
+openssl aes-256-cbc -md md5 -d -in exported.key.enc -out $KEY_FILENAME -k $OPENSSL_KEY
```

Comment:
md5 whyyyyyy

Reply:
The AMI we're using has a higher version of openssl than circleCI's. In the version switch the default method changed so to stay consistent we have to specify it now. Specifying it doesn't break older versions though so we don't have to worry about backwards compatibility.